crates/control/Cargo.toml

```diff
@@ -7,6 +7,7 @@ homepage = "https://github.com/hulks/hulk"
 
 [dependencies]
 approx = { workspace = true }
+byteorder = { workspace = true }
```

Comment:
Make sure to rebase, this dependency was removed from the webots falling branch before it was merged as it is unused

Reply:
Done